### PR TITLE
docs: updated link documentation

### PR DESCRIPTION
**Pull request by jessmetivier**
## Description
 _Updated **Link** docs_

### CODE REVIEW

**Comment by salarenko** [comment]
Whats up with that `{{ }}` description notation? I've seen it in previous PRs.

File: stories/link/link.stories.js [hunk 1]
```diff
@@ -3,38 +3,76 @@ import '../../dist/link.css';
...
-        iframeHeight: 300
+        iframeHeight: 300,
+        storyDescription: `There are different types of links for various use cases.
+    
+| Types | Modifier class | Use to... |
+| ----:| :--------------- | :--------------|
+| Default | *n/a* | Display a simple link. |
+| Subtle | <code>fd-link--subtle</code> | Distinguish between important (default) and less important (subtle) links in tables with large data lists. |
+| Disabled | <code>fd-link--disabled</code> | Display a link that a user cannot interactive with. |
+| Inverted | <code>fd-link--inverted</code> | Display a link in a **Shell Bar**. |
```
Comment (salarenko) [inline stories/link/link.stories.js:66]: Just to confirm, do we seriously have separate styling for Link, just to display it specificly in the `Shell Bar` 😄?
@InnaAtanasova @JKMarkowski

File: stories/link/link.stories.js [hunk 2]
```diff
@@ -3,38 +3,76 @@ import '../../dist/link.css';
 export default {
     title: 'Components/Link',
     parameters: {
-        description: 'Link component supports the use of modifiers, as well as the <code>is-visited</code> flag, which allows for optimal customization of your hyperlink. There are currently no restrictions as to which content can be included in this component.',
+        description: `A link (also known as hyperlink) is an interactive text element that allows users to navigate to another page or trigger a certain event.
+
+Note: Link supports the <code>is-visited</code> flag, which allows for optimal customization.
```
Comment (salarenko) [inline stories/link/link.stories.js:8]: This note doesn't say anything.
`is-visited` class can be used to manually style links as "visited" if HTML element or the browser doesn't support `:visited` state.

File: stories/info-label/info-label.stories.js [hunk 3]
```diff
@@ -80,4 +108,11 @@ export const rtl = () => `
     <span class="fd-info-label fd-info-label--accent-color-2 fd-info-label--icon sap-icon--upload-to-cloud"></span>
 </div>
 `;
-rtl.storyName = 'Info Label RTL';
+rtl.storyName = 'RTL';
+rtl.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: `The info label can be displayed from right to left on the screen.
```
Comment (salarenko) [inline stories/info-label/info-label.stories.js:115]: https://github.com/SAP/fundamental-styles/pull/1529#discussion_r483456136

**Review by salarenko — CHANGES_REQUESTED** [review]
This PR also contains `Info Label` documentation changes.

File: stories/link/link.stories.js (hunk 1 above)
Comment (InnaAtanasova) [reply stories/link/link.stories.js:64]: Screen Shot 2020-09-10 at 12 53 39 PM

Yes, according to the specs :)

**Comment by InnaAtanasova** [comment]
> This PR also contains `Info Label` documentation changes.

The file has been removed.

**Review by InnaAtanasova — APPROVED** [review]

**Review by salarenko — APPROVED** [review]

**Review by stefanoScalzo — APPROVED** [review]
LGTM

**Pull request merged**